node_modules/oae-core/signin/js/signin.js

```diff
@@ -0,0 +1,196 @@
+/*!
```

Comment:
When the URL is `http://cam.oae.com`, I am redirected to `/` after signing in instead of `/me`. Given that we are changing the me page location to `/`, this is probably fine, but I wanted to raise it in case it was indicative of a bug somewhere